webserver/views/datasets.py

```diff
@@ -142,9 +145,14 @@ def _parse_dataset_csv(file):
 @login_required
 def edit(dataset_id):
     ds = get_dataset(dataset_id)
+    pending = "0"
     if ds["author"] != current_user.id:
         raise Unauthorized("You can't edit this dataset.")
-
+    jobs = db.dataset_eval.get_jobs_for_dataset(str(dataset_id))
+    for job in jobs:
+        if(job["status"] in ("running","pending")):
```

Comment:
There also should be a space after a comma. If you are wondering about how to format Python code, check this website: http://pep8.org/.